google/cloud/bigtable/internal/instance_admin.h

```diff
@@ -178,6 +178,57 @@ class InstanceAdmin {
       bigtable::InstanceId const& instance_id,
       bigtable::ClusterId const& cluster_id, grpc::Status& status);
 
+  /**
+   * Makes an asynchronous request to get the attributes of a cluster.
+   *
+   * @param instance_id the id of the instance in the project, from that cluster
+   *     to be retrieved
+   * @param cluster_id the id of the cluster in the project that to be
+   *     retrieved.
+   * @param cq the completion queue that will execute the asynchronous calls,
+   *     the application must ensure that one or more threads are blocked on
+   *     `cq.Run()`.
+   * @param callback a functor to be called when the operation completes. It
+   *     must satisfy (using C++17 types):
+   *     static_assert(std::is_invocable_v<
+   *         Functor, google::bigtable::v2::MutateRowResponse&,
```

Comment:
Probably not a `MutateRowResponse`?  I think that is `google::bigtable::admin::v2::Cluster`?